openqa-desktop-gnome-auth-required-20230724.json

```diff
@@ -12,8 +12,6 @@
   "properties": [],
   "tags": [
     "ENV-VERSION-Tumbleweed",
-    "generic-desktop",
-    "openqa-desktop",

```

Comment:
why do you consider those invalid?

Reply:
I was thinking "this is not the desktop in the sense of unlocked and ready to use" at first. On the other hand [ensure_unlocked_desktop](https://github.com/os-autoinst/os-autoinst-distri-openQA/blob/master/lib/utils.pm#L79) is expecting `generic-desktop` to be a locked screen under GNOME?

Reply:
Because 'generic-desktop' is unlocked desktop ready for user interaction.

Reply:
More exactly https://github.com/os-autoinst/os-autoinst-distri-openQA/blob/master/lib/utils.pm#L79 has while loop when on start checks needles .. and if is `generic-destkop` preset it will quit loop at the end. But after this three needles can be shown the lock screen password prompt -> it needs the next iteration of the loop to handle it, but because of this tag -> loop is exited and the test continues with start_x11_app or any other test code